frontend/tests/email/maildev-mailbox.ts

```diff
@@ -21,9 +23,23 @@ export class MaildevMailbox extends Mailbox {
   }
 
   async fetchMyEmails(): Promise<MaildevEmail[]> {
-    // Maildev REST API docs: https://github.com/maildev/maildev/blob/master/docs/rest.md
-    const response = await this.api.get(`http://localhost:1080/email`);
-    const emails = await response.json() as MaildevEmail[];
-    return emails.filter(email => email.to.some(to => to.address === this.email));
+    const MAX_TRIES = 3;
+
+    for (let tries = 1; tries <= MAX_TRIES; tries++) {
+      try {
+        // Maildev REST API docs: https://github.com/maildev/maildev/blob/master/docs/rest.md
+        const response = await this.api.get(`http://localhost:1080/email`);
```

Comment:
Maybe we should also just use fetch as there's no reason we need to do this through the browser context.

Reply:
I guess we could, yeah. I'm not sure what the advantage would be though. I don't expect the errors are causing by using the browser context. Playwright trace wise...there's probably a difference and we monitor the browser context for unexpected response codes.

Anyway...I don't see a strong argument for changing this 🤷.